src/apps/inventory/models.py

```diff
@@ -35,6 +36,7 @@ class Product(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
     slug = AutoSlugField(populate_from="name", unique=True)
     is_active = models.BooleanField(default=True)
+    stock = models.PositiveIntegerField(default=1, validators=[MinValueValidator(0)])
```

Comment:
Why default stock is 1? What if we create multiple draft objects and wait for backoffice to deliver items to my store location.

Reply:
I was hesitating between 1 and 0, I think in your case the 0 is better. I was just thinking that when seller is creating a product it would be more intuitive to automatically have it in all products list.